.env.docker.development.example

```
@@ -14,8 +14,7 @@ DB_DECIMAL_NUMBERS=true
 PUBLIC_URL=http://localhost:18055
 DASH_URL=http://localhost:13010
 LOG_LEVEL=trace
-CORS_ENABLED=true
-CORS_ORIGIN=http://localhost:13000,http://localhost:13010
+CORS_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORS_ENABLED key should go before the DASH_URL key

(UnorderedKey)